.jenkins/dsl/jobs.groovy

```diff
@@ -107,27 +109,35 @@ void setupDeployJob(String jobFolder, KogitoJobType jobType) {
             stringParam('IMAGE_TAG', '', 'Image tag to use to deploy images')
 
             // Test config if needed specifics. Else test default config will apply.
-            booleanParam('KOGITO_IMAGES_IN_OPENSHIFT_REGISTRY', false, 'Set to true if kogito images for tests are in internal Openshift registry.In this case, KOGITO_IMAGES_REGISTRY and KOGITO_IMAGES_NAMESPACE parameters will be ignored')
+            booleanParam('KOGITO_IMAGES_USE_OPENSHIFT_REGISTRY', false, 'Set to true if kogito images for tests are in internal Openshift registry.In this case, KOGITO_IMAGES_REGISTRY and KOGITO_IMAGES_NAMESPACE parameters will be ignored')
             stringParam('KOGITO_IMAGES_REGISTRY', "${CLOUD_IMAGE_REGISTRY}", 'Test images registry')
             stringParam('KOGITO_IMAGES_NAMESPACE', "${CLOUD_IMAGE_NAMESPACE}", 'Test images namespace')
             stringParam('KOGITO_IMAGES_NAME_SUFFIX', '', 'Test images name suffix')
             stringParam('KOGITO_IMAGES_TAG', '', 'Test images tag')
+
             stringParam('EXAMPLES_URI', '', 'Git uri to the kogito-examples repository to use for tests.')
             stringParam('EXAMPLES_REF', '', 'Git reference (branch/tag) to the kogito-examples repository to use for tests.')
-
-            // Release information
-            stringParam('PROJECT_VERSION', '', 'Optional if not RELEASE. If RELEASE, cannot be empty.')
+            stringParam('EXAMPLES_IMAGES_CACHE_MODE', 'never', 'Set the examples images\' cache mode for the BDD tests. Default it will build the image if none existing.')
```

Comment:
```suggestion
            stringParam('EXAMPLES_IMAGES_CACHE_MODE', 'never', 'Set the examples images\' cache mode for the BDD tests. Default it will always build the image.')
```

Reply:
I suppose the default is meant the job default.